### ISSUE TITLE
Default table view for Get-WingetPackage should fit all properties in 120 char screen

### ISSUE THREAD

**Issue by peetrike** [microsoft/winget-cli#3466]
### Description of the new feature / enhancement

The default table view of `Get-WingetPackage` does not specify column with for **Name** and **Id** columns.  Because of that, some other fields might not fit in 120-symbols width window.

### Proposed technical implementation details

Add fixed width to Name and ID properties.

**Comment by dftf-stu** [comment]
Yeah, even in the CMD version, it's pretty-bad how essential info gets cut-off:

<img width="1366" height="738" alt="Image" src="https://github.com/user-attachments/assets/ac6a2347-82aa-445a-a878-2d6d9bbfc582" />

Good-luck a person finding the Package ID for the language they may want for the Firefox Developer Edition from that listing!
